common/app/views/fragments/immersiveGalleryHeadTonal.scala.html

```diff
@@ -0,0 +1,90 @@
+@(item: model.ContentType, showBadge: Boolean = false, showMeta: Boolean = true)(implicit request: RequestHeader)
+
+@import views.support.TrailCssClasses.toneClass
+@import views.support.ContributorLinks
+
+<header class="content__head content__head--immersive content__head--desktop is-fixed-height tonal__head tonal__head--@toneClass(item)">
```

Comment:
yeah I mean your first paragraph, I feel like the group of semi-common stuff is as big as the actual common stuff, so it makes sense to keep it separate.  However your PR is probably not the right place to do too much!  As for this file, I haven't investigated how much is in common with the other file, but if it's more common than not it makes sense to combine them.